wire-ios-sync-engine/Source/Calling/AVS/AVSWrapper.swift

```diff
@@ -393,14 +393,20 @@ public final class AVSWrapper: AVSWrapperType {
 
     private let requestClientsHandler: Handler.RequestClients = { handle, conversationIdRef, contextRef in
         AVSWrapper.withCallCenter(contextRef, conversationIdRef) { (callCenter, conversationId: String) in
-            let completion: (String) -> Void = { (clients: String) in
-                wcall_set_clients_for_conv(handle, conversationIdRef, clients)
-            }
 
             // This handler is called once per call, but the participants may be added or removed from the
             // conversation during this time. Therefore we store the completion so that it can be re-invoked
             // with an updated client list.
             let conversationId = AVSIdentifier.from(string: conversationId)
+
+            let completion: (String) -> Void = { (clients: String) in
+                callCenter.setMLSConferanceInfoIfNeeded(for: conversationId) { isMLSConversation in
+                    if !isMLSConversation {
+                        wcall_set_clients_for_conv(handle, conversationIdRef, clients)
+                    }
```

Comment:
So is the flow `when requested for clients, if the conference is mls, then set the conference info, otherwise fetch and pass the list of clients`?

Reply:
yes